manifests/010-metricservice.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: samples-operator-tls
+  labels:
+    app: samples-operator
+  name: metrics
+  namespace: openshift-cluster-samples-operator
+spec:
+  ports:
+  - name: https
+    port: 443
+    protocol: TCP
+    targetPort: 8443
```

Comment:
Aren't you using a different port for the metrics server?

Reply:
yeah this stemmed from copying from OCM-O in trying to get it to work, but yeah, updating this to match the port the golang code is doing might be at least part of my problem re: getting openshift-monitor pods to fetch my stats

Reply:
switched this back .... things are not quite working but some progress has been made ... will post the latest status in a non-review comment.